constrain/Classes/Modification.swift

```diff
@@ -30,6 +30,17 @@ extension Constraints {
         return constraints[identifier]
     }
     
+    @discardableResult
+    public func removeConstraintWithIdentifier(_ identifier: ConstraintIdentifier) -> Self {
+        guard let constraint = layoutConstraintWithIdentifier(identifier) else { return self }
+        view?.removeConstraint(constraint)
+        constraints[identifier] = nil
+        if let index = allConstraints.firstIndex(of: constraint) {
+            allConstraints.remove(at: index)
+        }
```

Comment:
Maybe use Array.removeAll(where:) instead

Reply:
Would make sense, but layoutConstraintWithIdentifier only returns one constraints. Let me refactor so it does all of them